pkg/config/server/server.go

```diff
@@ -313,10 +318,6 @@ type SecurityCheckConfigFile struct {
 type LimitConfigFile struct {
 	DefaultTenantRetentionPeriod string `mapstructure:"defaultTenantRetentionPeriod" json:"defaultTenantRetentionPeriod,omitempty" default:"720h"`
 
-	DefaultWorkflowRunLimit      int           `mapstructure:"defaultWorkflowRunLimit" json:"defaultWorkflowRunLimit,omitempty" default:"1000"`
```

Comment:
i think we have this in our docs for OSS so this is technically a breaking change if someone has a workflow run limit limit set but not a task run limit set. 

perhaps we should also set TR limit to WRF env var in this case?